amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/InterchangeEndpoints.java

```diff
@@ -238,6 +241,99 @@ public Response getValues(List<String> fields) {
         }
         return Response.ok(response, responseType).build();
     }
+    
+    /**
+     * Returns a list of values for all id of requested fields.
+     * 
+     * For fields like locations, sectors, programs the object contains the ancestor values.
+     * <h3>Sample request:</h3><pre>
+     * [
+     *   "locations~location": [534, 126],
+     *   "national_plan_objective~program": [123],
+     *   "primary_sectors~sector": [297]
+     * ]
```

Comment:
please correct the output that should be 
`
{"primary_sectors~sector":[150,151],"locations~location":[51,61],"national_plan_objective~program":[70]}
`
instead of `["primary_sectors~sector":[150,151],"locations~location":[51,61],"national_plan_objective~program":[70]]`